.github/workflows/build-publish.yml

```diff
@@ -10,31 +10,27 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
+    - name: Checkout code 🛒
+      uses: actions/checkout@v4
+
     - name: Update version file ⬆️
       uses: brettdorrans/write-version-to-file@v1.1.0
```

Comment:
if we can use the git tag as a source of version, we can get rid of this step and the .VERSION file.